dev/ParallaxView/ScrollInputHelper.h

```diff
@@ -11,17 +11,17 @@ class ScrollInputHelper
         std::function<void(bool, bool)> infoChangedFunction);
     ~ScrollInputHelper();
 
-    winrt::UIElement TargetElement() const;
-    winrt::CompositionPropertySet SourcePropertySet() const;
-    bool IsTargetElementInSource() const;
-
-    winrt::hstring GetSourceOffsetPropertyName(winrt::Orientation orientation) const;
-    winrt::hstring GetSourceScalePropertyName() const;
-    double GetOffsetFromScrollContentElement(const winrt::UIElement& element, winrt::Orientation orientation) const;
-    double GetMaxUnderpanOffset(winrt::Orientation orientation) const;
-    double GetMaxOverpanOffset(winrt::Orientation orientation) const;
-    double GetContentSize(winrt::Orientation orientation) const;
-    double GetViewportSize(winrt::Orientation orientation) const;
+    [[nodiscard]] winrt::UIElement TargetElement() const;
+    [[nodiscard]] winrt::CompositionPropertySet SourcePropertySet() const;
```

Comment:
i'm surprised to see this in the header but not a corresponding change to a body?